ownCloudAppShared/AppLock/PasscodeViewController.swift

```diff
@@ -21,7 +21,7 @@ import UIKit
 public typealias PasscodeViewControllerCancelHandler = ((_ passcodeViewController: PasscodeViewController) -> Void)
 public typealias PasscodeViewControllerCompletionHandler = ((_ passcodeViewController: PasscodeViewController, _ passcode: String) -> Void)
 
-public class PasscodeViewController: UIViewController, Themeable {
+public class PasscodeViewController: UIViewController, Themable {
```

Comment:
Both spellings are sprinkled throughout this codebase. I don't have a particular preference